### ISSUE TITLE
Full AI review API returns branches without moves

### ISSUE THREAD

**Issue by mmartin** [online-go/online-go.com#1316]
**Describe the bug**
Sometimes (usually at the end during passing) the full AI review API returns branches without moves, e.g.
```
"move-244": {
...
    "branches": [
...
        {
            "moves": [],
            "win_rate": 0.9987,
            "score": 8.52,
            "visits": 138,
            "lcb": 0.9976,
            "score_mean": 8.52,
            "score_stdev": 1.5283,
            "utility": 1.0349,
            "utility_lcb": 1.0271
        },
...
```

It breaks AI review if it is one of the first 6 branches:
https://github.com/online-go/online-go.com/blob/a90e0bcba6237554fda0fd7d3d46121d551a26c2/src/views/Game/AIReview.tsx#L391

Exception is thrown on line 432 (since `mv` is `undefined`):
https://github.com/online-go/online-go.com/blob/a90e0bcba6237554fda0fd7d3d46121d551a26c2/src/views/Game/AIReview.tsx#L428-L434

I guess, one of the solutions could be to replace line 391 with –
```ts
let branches = ai_review_move.branches.filter(branch => branch.moves.length > 0).slice(0, 6);
```

**Comment by anoek** [comment]
Can you provide a link to a game with this bug by chance? You can send it to anoek@online-go.com if you don't want it here for whatever reason. Thanks!

**Comment by anoek** [comment]
This was fixed in #1569
